File: 프로그래머스/0/120585. 머쓱이보다 키 큰 사람/머쓱이보다 키 큰 사람.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class Solution 
{
    public int solution(int[] array, int height) 
    {
        
        var tollerList = array.Where(w => w > height)
                                .ToArray();
                                
        
        return tollerList.Length;
    }
}

File: 프로그래머스/0/120585. 머쓱이보다 키 큰 사람/머쓱이보다 키 큰 사람.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<int> array, int height) 
{
    array.push_back(height);
    sort(array.begin(), array.end());
    int index = 0;
    for(int i=0; i<=array.size(); i++)
    {
        if(array[i] == height)
        {
            index = i;
        }
    }
    return array.size() - index - 1;
}